post_office/utils.py

```diff
@@ -138,14 +138,22 @@ def parse_emails(emails):
     return emails
 
 
-def cleanup_expired_mails(cutoff_date, delete_attachments=True):
+def cleanup_expired_mails(cutoff_date, delete_attachments=True, batch_size=10000):
```

Comment:
Thinking about this, can you make this lower to 1k?

@jrief is also right, do you mind adding a `--batch-size` argument to the management command also?